src/js/game/hud/parts/statistics_handle.js

```diff
@@ -31,8 +32,24 @@ export class HUDShapeStatisticsHandle {
 
     initElement() {
         this.element = document.createElement("div");
-        this.element.setAttribute("data-shape-key", this.definition.getHash());
-
+        const shape = this.definition.getHash();
+        this.element.setAttribute("data-shape-key", shape);
+        const pinButton = document.createElement("button");
+        pinButton.classList.add("pin");
+        const pinDetector = new ClickDetector(pinButton, {
```

Comment:
Minor nitpick: If you create a clickDetector, you have to make sure to call cleanup(). I think in destroy() would be a good place (Basically whenever you call this.element.remove())

Reply:
Thanks